specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2023-09-15-preview/dataTransferService.json

```diff
@@ -0,0 +1,547 @@
+          },
+          {
+            "$ref": "#/parameters/JobNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful Cancel Job response",
+            "schema": {
+              "$ref": "#/definitions/DataTransferJobGetResults"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "cosmos-db.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}/dataTransferJobs": {
+      "get": {
+        "operationId": "DataTransferJobs_ListByDatabaseAccount",
+        "description": "Get a list of Data Transfer jobs.",
+        "x-ms-examples": {
+          "CosmosDBDataTransferJobFeed": {
+            "$ref": "./examples/data-transfer-service/CosmosDBDataTransferJobFeed.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./cosmos-db.json#/parameters/accountNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "All Data Transfer Jobs retrieved successfully",
+            "schema": {
+              "$ref": "#/definitions/DataTransferJobFeedResults"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "cosmos-db.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "DataTransferDataSourceSink": {
+      "type": "object",
+      "description": "Base class for all DataTransfer source/sink",
+      "properties": {
+        "component": {
+          "type": "string",
+          "enum": [
+            "CosmosDBCassandra",
+            "CosmosDBMongo",
+            "CosmosDBSql",
+            "AzureBlobStorage"
+          ],
+          "x-ms-enum": {
+            "name": "DataTransferComponent",
+            "modelAsString": true
+          },
+          "default": "CosmosDBCassandra"
+        }
+      },
+      "discriminator": "component",
+      "required": [
+        "component"
+      ]
+    },
+    "CosmosDataTransferDataSourceSink": {
```

Comment:
We cannot rename to DataTransferDataSourceSink to CosmosDataTransferDataSourceSink as it is generic and not specific to Cosmos. Also this is not a new change added to this version, so any change like this would be a breaking change.